### ISSUE TITLE
Synchronous parser.parse returns empty string literals

### ISSUE THREAD

**Issue by m0ru** [rdfjs/N3.js#136]
```js
var parser = N3.Parser({ format: 'application/trig' });
var rdf = 
`@prefix dc: <http://purl.org/dc/elements/1.1/>.
@prefix ex: <http://example.org/>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.

ex:1234 dc:description "hello world!"^^xsd:string.`
parser.parse(rdf)
```
returns an empty object value

```js
[
  {
    "subject": {
      "termType": "NamedNode",
      "value": "http://example.org/1234"
    },
    "predicate": {
      "termType": "NamedNode",
      "value": "http://purl.org/dc/elements/1.1/description"
    },
    "object": {
      "termType": "Literal",
      "value": "", // <-- should be "hello world!" afaict
      "language": "",
      "datatype": {
        "termType": "NamedNode",
        "value": "http://www.w3.org/2001/XMLSchema#string"
      }
    },
    "graph": {
      "termType": "DefaultGraph",
      "value": ""
    }
  }
]
```

**version:** 1.0.0-alpha

Tried step-debugging it but I have not the slightest idea what's going on in the parser and why this might happen. sry ^^"

We're using the async version atm anyway. I just wanted to try the synchronous version to see if we can switch, as this would allow us to pull the parsing into the side-effect-free redux-reducer, amongst others. Staying with the async variant works for now as well :)

**Comment by RubenVerborgh** [comment]
Should indeed be fixed by ba4f74827cc84a9072d87635529f9b96d60711e4.

**Comment by RubenVerborgh** [comment]
Released as 1.0.0-beta.1
